XX.md

```diff
@@ -0,0 +1,35 @@
+NIP-XX
+======
+
+External Content IDs
+-------------------------
+
+`draft` `optional`
+
+There are certain established global content identifiers that would be useful to reference in nostr events so that clients can query all events assosiated with these ids.
+
+- Book [ISBNs](https://en.wikipedia.org/wiki/ISBN)
+- Podcast [GUIDs](https://podcastnamespace.org/tag/guid)
+- Movie [ISANs](https://en.wikipedia.org/wiki/International_Standard_Audiovisual_Number)
+
+Since the `i` tag is already used for similar references in kind-0 metadata events it makes sense to use it for these content ids as well.
+
+
+## Supported IDs
+
+### Books:
+
+- Book ISBN: `["i", "book:isbn:9780765382030"]` - https://isbnsearch.org/isbn/9780765382030
```

Comment:
I dont think it makes sense to prefix the id with the media, it should just be prefixed with the database type ie `isbn:watever` because in some cases those databases contain all kinds of media

Reply:
@v0l yes that's a great point - I've updated the `isbn` and `isan` examples to remove the media prefix - https://github.com/nostr-protocol/nips/pull/1185/commits/d33b223c6afe2be3d197d2cdb4686961a6fb2b29

For the `podcast:guid` case I've left it in because `podcast:guid` is equivalent to `isbn` in denoting the namespace that is used.

Reply:
It is useful to have a 2nd level for those other media types, this is what im doing in the Torrent NIP, i have `ttvdb:series:123` and `ttvdb:movies:123` because those sets can overlap, if books or other media overlaps the ids then you basically need this.

Reply:
I think for the `isbn` and `isan` there will never be any overlap